addons/binding/org.openhab.binding.dsmr/README.md

```diff
@@ -0,0 +1,294 @@
+# DSMR Binding
+## Introduction
+The DSMR-binding is targeted towards Dutch users having a smart meter (Dutch: 'Slimme meter'). Data of Dutch smart meters can be obtained via the P1-port. When connecting this port from a serial port the data can be read out.
+
+This binding reads the P1-port of the Dutch Smart Meters that comply to NTA8130, DSMR v2.1, DSMR v2.2, DSMR v3.0, DSMR v4.0, DSMR v4.04, and DSMR 5.0.  
+Although DSMR v4.2 is not an official specification, the binding has support for this version.
+
+If you are not living in the Netherlands but do want to read a meter please have look at the [IEC-62056-21 Meter Binding](https://github.com/openhab/openhab1-addons/wiki/IEC-62056---21-Meter-Binding).
```

Comment:
I'm not the maintainer of this binding, so i don't know if it is ported already, or if it is WIP.

The link is correct (i.e. it refers to valid page) and i think we need to have something here since as far as i know only people in the Netherlands do have equipment that will be working with this binding. If there is already an OH2 implementation of this generic meter binding please let me know. I will update the URL.